packages/lib/shim-init-node.js

```diff
@@ -9,10 +9,12 @@ const FsDriverNode = require('./fs-driver-node').default;
 const mimeUtils = require('./mime-utils.js').mime;
 const Note = require('./models/Note').default;
 const Resource = require('./models/Resource').default;
+const Setting = require('./models/Setting').default;
```

Comment:
I don't think we'd want to add this dependency here, to avoid circular references. Please use dependency injection by passing the parameters as shimInit({ proxyUrl: ...., proxyTimeout:.... })`.